server/handler.go

```diff
@@ -35,9 +34,36 @@ func InternalSubscriber(b bool) SubscriberOption {
 	}
 }
 
+type SubscriberOption func(*SubscriberOptions)
+
+func NewSubscriberOptions(opts ...SubscriberOption) SubscriberOptions {
+	opt := SubscriberOptions{
+		Context: context.Background(),
+	}
+
+	for _, o := range opts {
+		o(&opt)
+	}
+
+	return opt
+}
+
+type SubscriberOptions struct {
```

Comment:
Types should remain at the top. Only move NewSubscriberOptions.

Reply:
ok